implementations/rust/ockam/ockam_app/src/shared_service/tray_menu.rs

```diff
@@ -36,11 +37,19 @@ pub(crate) async fn build_shared_services_section(
 
 /// Event listener for the "Create..." menu item
 pub fn on_create(app: &AppHandle<Wry>) -> tauri::Result<()> {
-    tauri::WindowBuilder::new(
+    let w = tauri::WindowBuilder::new(
```

Comment:
Have you by any chance identified the syntax needed to either get an existing window by ID, then fall back to create it on the fly as you do here? I've only done minimal research because "show existing window" and "create net-new window" are both straightforward, but for this and my invitations use-case I think not erroring if the window is already displayed, just bringing it into the foreground or marking it as always-on-top, would be an improvement.

Reply:
I've done some changes and will look like this:
```rust
match app.get_window(SHARED_SERVICE_WINDOW_ID) {
    None => .. // Create from scratch
    Some(w) => w.show()?,
}
```

Is that what you mean?

Reply:
Yeap, exactly, thank you.